css/components/blog.css

```diff
@@ -0,0 +1,109 @@
+.row.blog {
```

Comment:
not sure ka sitas daro? Atrodo, kad net isstumia viska labiau i sonus, uz grido ribu